### ISSUE TITLE
use a font where "l" and "I" don't look exactly the same

### ISSUE THREAD

**Issue by computron** [materialsintelligence/matstract#162]
e.g. do a search for "ferroelectric" but not "perovskite".

You'll see TlInS2

This is T-L-I-N-S-2

But the L and I look the same.

Unfortunately many fonts use the same vertical line for lowercase L and uppercase I ...

http://www.crossbarifonts.info

@mkhorton not your problem but nevertheless up your alley? lol

**Comment by mkhorton** [comment]
Yup, had a conversation about this over on materials_django https://github.com/materialsproject/materials_django/issues/461

Donny switched to Roboto Slab, but since then IBM has release a free and open font, IBM Plex, which I think is much nicer -- it's still a Sans Serif font rather than a Slab, but properly disambiguates I, l etc.

**Comment by mkhorton** [comment]
If you want a search term for this, apparently you're looking for a "textbook" font
